src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -736,28 +819,28 @@ protected void addPublicIpWithNewSecondaryNetworkInterface(Azure azureService, V
                      ex);
             handleAddPublicIpWithNewSecondaryNetworkInterfaceException(azureService,
                                                                        vm,
-                                                                       publicIpAddress,
+                                                                       PublicIPAddress,
                                                                        newSecondaryNetworkInterface);
         }
     }
 
     protected void handleAddPublicIpWithNewSecondaryNetworkInterfaceException(Azure azureService, VirtualMachine vm,
-            PublicIpAddress publicIpAddress, NetworkInterface newSecondaryNetworkInterface) {
+            PublicIPAddress PublicIPAddress, NetworkInterface newSecondaryNetworkInterface) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)